app/templates/app/project/issue/index.hbs

```diff
@@ -182,8 +182,8 @@
                     </td>
                     <td data-field="issuetype.name" class="issue-type">{{issue.issuetype.name}}</td>
                     <td data-field="issue.milestone" class="issue-milestone"><a href="#">{{issue.issuemilestone.name}}</a></td>
-                    <td data-field="issue.startDate" class="issue-start">{{moment-format issue.startDate 'DD/MMM/YY'}}</td>
-                    <td data-field="issue.endDate" class="issue-end">{{moment-format issue.endDate 'DD/MMM/YY'}}</td>
+                    <td data-field="issue.startDate" class="issue-start">{{format-date date=issue.startDate format="DD MMM 'YY"}}</td>
```

Comment:
Weren't we going to decide the format and store that in a service to be used all across the application?

Reply:
Ok so the default format is stored in the helper now and if someone wants to use some specific format then they can pass format as an argument to that helper. Otherwise, the default format will be used across the application.